suite-common/graph/src/graphDataFetching.ts

```diff
@@ -262,12 +271,21 @@ export const getMultipleAccountBalanceHistoryWithFiat = async ({
                     fiatCurrency,
                     forceRefetch,
                     isElectrumBackend,
-                }).then(res => {
-                    if (res === null)
-                        throw new Error(`Unable to fetch fiat rates for defined timestamps.`);
-
-                    return [coin, res] as const;
-                }),
+                })
+                    .then(res => {
+                        if (res === null)
+                            throw new Error(`Unable to fetch fiat rates for defined timestamps.`);
```

Comment:
can we add coin and fiat currency here as well? Similar to what you do in the catch clause?

Reply:
But actually what do you think?

Reply:
I don't think having fiat currency here is useful since for this fetch it just doesn't matter.